src/main/resources/application.properties

```diff
@@ -6,3 +6,14 @@ spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
 spring.jpa.hibernate.ddl-auto=create-drop
 
+spring.mail.host=localhost
```

Comment:
It is required to install something else on your local to set up the mail server?

Reply:
It is required to use "npm install -g maildev"